### ISSUE TITLE
setUserEmail for Android

### ISSUE THREAD

**Issue by adrielschmitz** [walterholohan/react-native-crisp-chat-sdk#10]
When trying to use the setUserEmail method on the android platform, I realized that the email was not set. Looking at the dependencies I realized that there is a condition to only set the email if it is in IOS ```if (Platform.OS === 'ios')```.
I thought about using CrispChatSDK.setUserEmail, which was already done in the old version. However, I get an error that says "undefined is not a function".
![2020-11-18T15_24_55 117Z](https://user-images.githubusercontent.com/42657151/99550131-2278a880-2999-11eb-8bc0-d4a80e050808.jpeg)

**Comment by walterholohan** [comment]
Hey @adrielschmitz, yes with the new release 0.3.0 I am using Crisp's new android SDK which is still in beta version but has not exposed any of the API methods such as setUserEmail(). But they plan on releasing that functionality. So, for now, setUserEmail() is only available on iOS. I have noted that in the README but will try call it out more so people don't miss it.

**Comment by adrielschmitz** [comment]
Understand. I'll have to wait for them to release then. Do you think I keep the issue open to serve as a warning to other people?

**Comment by walterholohan** [comment]
Yep please do. The minute Crisp release a new android version I will update this library.

**Comment by 4rno** [comment]
Hi, a little update for this thread,

@walterholohan it seems that Crisp released 2 days ago a new android version of the SDK (available on master), and it is now possible to use methods like `setUserEmail` and `setUserNickname`.

I launched their demo and added both methods in the `Application.java`. And it works.

```
[...]
Crisp.configure(getApplicationContext(), "7598bf86-9ebb-46bc-8c61-be8929bbf93d");
Crisp.setUserEmail("john@doe.com");
Crisp.setUserNickname("John Doe");
[...]
```

**Comment by walterholohan** [comment]
Thanks @4rno for the heads up. Hopefully ill get time over the coming days to upgrade and use the latest android sdk from Crisp

**Comment by walterholohan** [comment]
@4rno @adrielschmitz I just released version 0.6.0 which has the updated Android SDK and some new API methods

**Comment by 4rno** [comment]
@walterholohan Thank you so much, I will integrate the 0.6.0 in my project on monday !
